source/lib-intro.tex

```diff
@@ -3606,7 +3559,7 @@
 in~\ref{global.functions}, member functions in~\ref{member.functions}, data race
 avoidance in~\ref{res.on.data.races}, access specifiers
 in~\ref{protection.within.classes}, class derivation in~\ref{derivation}, and
-exceptions in~\ref{res.on.exception.handling}.
+exceptions in~\ref{res.on.exception.handling}.  %JMB: This intro fluff should add "and contract assertions in~\ref{res.contract.assertions}  Also, the "its use" doesn't seem right in this sentence.   Can I make this a bulleted list so that it resembles a sentence again?
```

Comment:
Please drop the JMB in a fix-up commit to the original one (will be squashed).
Then, add a separate commit (same pull request) with proper "[label] title" commit description that improves this (yes, a bulleted list seems great).  Do not silently improve this in the original paper-applying commit.

Reply:
done